src/Pillar-Tests-Cli/ClapPillarBuildCommandTest.class.st

```diff
@@ -0,0 +1,30 @@
+Class {
+	#name : #ClapPillarBuildCommandTest,
+	#superclass : #TestCase,
+	#category : #'Pillar-Tests-Cli'
+}
+
+{ #category : #adding }
+ClapPillarBuildCommandTest >> testAddSubcommand [
+	| allIds defaultCommand |
+	defaultCommand := ClapPillarBuildCommand defaultCommand.
+	allIds := PRTarget clapCommandList collect: [ :c | c identifier ].
+	allIds do: [ :id |
+		self assert: ((defaultCommand subcommands select: [ :c | c class = ClapCommand new class and: c identifier = id asSymbol ]) size = 1).
+		 ]
```

Comment:
Maybe you can use `#anySatify:`? And directly compare the command identifier. There is no need to create another command :)

Reply:
Solved!